presto-native-execution/pom.xml

```diff
@@ -159,18 +159,28 @@
                     </ignoredResourcePatterns>
                 </configuration>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
-                <artifactId>maven-surefire-plugin</artifactId>
-                <configuration>
-                    <forkCount>1</forkCount>
-                    <reuseForks>false</reuseForks>
-                    <systemPropertyVariables>
-                        <PRESTO_SERVER>/root/project/build/debug/presto_cpp/main/presto_server</PRESTO_SERVER>
-                        <DATA_DIR>/tmp/velox</DATA_DIR>
-                    </systemPropertyVariables>
-                </configuration>
-            </plugin>
         </plugins>
     </build>
+
+    <profiles>
+        <profile>
+            <id>presto-native-tests</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.apache.maven.plugins</groupId>
+                        <artifactId>maven-surefire-plugin</artifactId>
+                        <configuration>
+                            <includes>
+                                <include>**/com/facebook/presto/nativeworker/**/Test*.java</include>
+                            </includes>
+<!--                            <excludes>-->
```

Comment:
Why do we have this? Let's remove commented out section and consider running all tests.

Reply:
> Why do we have this? Let's remove commented out section and consider running all tests.

@mbasmanova That's because I got some spark-related test failures earlier so I wanted to see if can have a clean run w/o those tests. I will remove this after all things are green.